### PR TITLE
Curriculum app + tests

### PR DESCRIPTION
**Pull request by Byte4Craft**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTP-29)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [x] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/curriculum/models.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+    name = models.CharField(max_length=50, unique=True, blank=False)
+    goals = ArrayField(models.CharField(max_length=30, blank=False), size=8, null=True)
+    description = models.TextField(max_length=500, blank=True)
+    mentors = models.ManyToManyField(CustomUser, blank=False, related_name='mentors')
+    team = models.ForeignKey(Team, null=True, related_name='team')
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
```
Comment (skramar) [inline eventually/curriculum/models.py:48]: Rewrite str and repr methods

Comment (Byte4Craft) [reply eventually/curriculum/models.py:48]: Done.

File: eventually/curriculum/migrations/0002_auto_20171207_0153.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (skramar) [inline eventually/curriculum/migrations/0002_auto_20171207_0153.py]: Change migration name

File: eventually/curriculum/models.py [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+        |   'updated': 1511394690
+        | }
+        """
+
+        mentors = [mentor.id for mentor in self.mentors.all()] if self.mentors else None
+
+        return {'id:': self.id,
+                'name:': self.name,
+                'description': self.description,
+                'goals': self.goals,
```
Comment (skramar) [inline eventually/curriculum/models.py:183]: team should be an object

File: eventually/curriculum/migrations/0002_auto_20171207_0153.py (hunk 2 above)
Comment (Byte4Craft) [reply eventually/curriculum/migrations/0002_auto_20171207_0153.py]: Done.

File: eventually/curriculum/models.py (hunk 3 above)
Comment (Byte4Craft) [reply eventually/curriculum/models.py:183]: Done.

**Review by skramar — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**